packages/react/src/components/Link/Link.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import type { Meta, StoryFn, StoryObj } from '@storybook/react';
+
+import { Paragraph } from '../Typography';
+
+import { Link } from '.';
+
+type Story = StoryObj<typeof Link>;
+
+export default {
+  title: 'Kjernekomponenter/Link',
+  component: Link,
+  parameters: {
+    status: {
+      type: 'beta',
+      url: 'http://www.url.com/status',
+    },
+  },
+} as Meta;
+
+export const Normal: Story = {
+  args: {
+    children: 'Gå til designsystemet',
+    href: 'https://designsystemet.no/',
+  },
+};
+
+export const Invertert: Story = {
+  args: {
+    children: 'Gå til designsystemet',
+    inverted: true,
+    href: 'https://designsystemet.no/',
+  },
+  parameters: {
+    backgrounds: {
+      default: 'dark',
+    },
+  },
```

Comment:
We can use a decorator for this to get the dark preview in both docs and story mode!
```suggestion
  parameters: {
    layout: 'fullscreen',
  },
  decorators: [
    (Story) => (
      <div
        style={{
          background: '#333333',
          padding: '32px',
          justifyContent: 'center',
          display: 'flex',
        }}
      >
        <Story />
      </div>
    ),
  ],
```